py-polars/polars/internals/expr/expr.py

```diff
@@ -5621,40 +5621,39 @@ def ewm_var(
         alpha = _prepare_alpha(com, span, half_life, alpha)
         return wrap_expr(self._pyexpr.ewm_var(alpha, adjust, bias, min_periods))
 
-    def extend_constant(
-        self, value: int | float | str | bool | date | None, n: int
-    ) -> Expr:
+    def extend_constant(self, value: PythonLiteral | None, n: int) -> Expr:
         """
-        Extend the Series with given number of values.
+        Extremely fast method for extending the Series with 'n' copies of a value.
```

Comment:
Not sure the "Extremely fast" quantifier is necessary here, but I'll allow it 😄

Reply:
Haha... It really is though 🤣